demosplan/DemosPlanCoreBundle/Command/MaintenanceCommand.php

```diff
@@ -205,7 +206,7 @@ protected function initialize(InputInterface $input, OutputInterface $output)
      *
      * @throws ShutdownEndlessCommandException
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
@mussbach is there a fixer that also adjusts the doc-comment return types? I'd like to avoid merging new phpstan errors even if they are from automatic changes.